feature-group-definitions/dialog.yml

```diff
@@ -1,6 +1,15 @@
 spec: https://html.spec.whatwg.org/multipage/interactive-elements.html#the-dialog-element
 caniuse: dialog
 usage_stats: https://chromestatus.com/metrics/feature/timeline/popularity/481
+status:
+  baseline: false
+  support:
+    chrome: "37"
```

Comment:
Missing on Chrome Android? Is that really true?

Reply:
OK, so it's about the `cancel` event. https://mdn-bcd-collector.gooborg.com/tests/api/HTMLDialogElement doesn't have a test for that, I think because `oncancel` is on `GlobalEventHanders`.

I've checked the Chromium source and don't see anything special about the cancel event. I think BCD is wrong, can you separate out this change so we can fix BCD and proceed with the rest of `d*`?

Reply:
Dialog fix for BCD: https://github.com/mdn/browser-compat-data/pull/21459

Reply:
Refreshed the status with 728b8a2. Now rightfully Baseline low.